tsconfig.json

```
@@ -6,6 +6,6 @@
         "outDir": "./dist",
         "moduleResolution": "nodenext",
     },
-    "include": ["./src/**/*", "./test/**/*"],
+    "include": ["./src/**/*", "./test/**/*", "tests/account"],
```

should not touch tsconfig.json

```suggestion
    "include": ["./src/**/*", "./test/**/*"],
```